org.mwc.debrief.core/plugin.xml

```diff
@@ -1304,9 +1304,5 @@
             class="org.mwc.debrief.core.splashHandlers.ExtensibleSplashHandler"
             id="org.mwc.debrief.core.splashHandlers.extensible">
       </splashHandler>
```

Comment:
hmm, @ayeshare - isn't this removing the splash screen from the main Debrief app?

Reply:
oh i have accidentally committed this file.  let me undo it